src/reducer/noteReducer.js

```diff
@@ -0,0 +1,26 @@
+const initialNoteState = {
+    notes: [],
+    trash: [],
+    archivedNotes: []
+}
+
+const noteReducer = (state, { type, payload }) => {
+    switch (type) {
+        case "SET_INITIAL_NOTES":
+            return { ...state, notes: payload }
+        case "ADD_NEW_NOTE":
+            return { ...state, notes: payload }
+        case "ADD_NOTE_TO_TRASH":
+            return { ...state, trash: payload }
+        case "REMOVE_NOTE_FROM_NOTES":
+            return { ...state, notes: payload }
+        case "TOGGLE_PINNED_NOTES":
+            return { ...state, notes: payload }
+        case "REMOVE_FROM_TRASH":
+            return { ...state, trash: payload }
```

Comment:
Here you are returning the same thing `{ ...state, notes: payload }` multiple times. You can give a common name for it like `SET_NOTE` and for trash `SET_TRASH`. No need to name separate case for every action

Reply:
Changed it. Thanks for pointing out!